### DISCUSSION TITLE
Is this project still alive?

### DISCUSSION THREAD

**Discussion by lonelyadmin** [openargus/argus#7] [Q&A]
Just curious...

Used this for a project 10 years ago at $previous_job and need to do the same sort of thing.

**Comment by openargus** [comment]
Yes, although its been quiet recently !!Carter Carter Bullard • CEO250 E 53rd Street Suite 501New York, New York 10022Phone +1.212.588.9133 • Mobile +1.917.497.9494On Sep 20, 2023, at 7:30 PM, lonelyadmin ***@***.***> wrote:﻿
Just curious...
Used this for a project 10 years ago at $previous_job and need to do the same sort of thing.

**Reply by lonelyadmin** [reply]
Good to hear! The long quiet usually either means it's abandoned or it just works so well it hasn't needed to be changed!

**Reply by openargus** [reply]
I've been committed to other projects and I'm just now getting back to open source argus.
I'm going to fix anything that comes up that is a thread killer, but I'm working to release argus-4.0 very soon ... so stay tuned !!!
